flocker/cli/script.py

```diff
@@ -65,6 +65,12 @@ class DeployOptions(Options):
     optParameters = [
         ["port", "p", REST_API_PORT,
          "The REST API port on the server.", int],
+        ["cacert", None, None,
+         "Path to cluster certificate file."],
+        ["cert", None, None,
+         "Path to user certificate file."],
+        ["key", None, None,
+         "Path to user private key file."],
         ["certificates-directory", "c",
          None, ("Path to directory where TLS certificate and keys can be "
                 "found. Defaults to current directory.")],
```

Comment:
:question: Not currently part of this patch, but please change this to "Path to directory containing TLS certificates and keys. Defaults to current directory."